api/websocket/server/server.go

```diff
@@ -41,6 +41,7 @@ const (
 	pingInterval                 = 8 * time.Second
 	pongTimeout                  = 10 * time.Second // should be greater than pingInterval
```

Comment:
I think it's better to set  both `pongTimeout`  and `pingInterval` as Duration type, although they may have the same value.

Reply:
What do you mean? They are already `time.Duration` type now

Reply:
My bad, I misunderstood